src/main/java/com/cepengagementservice/Config/CorsConfig.java

```diff
@@ -5,21 +5,24 @@
 import org.springframework.web.servlet.config.annotation.CorsRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 
-    @Configuration
-    public class CorsConfig{
+/**
+ * Allows for CORS
+ * @author Unknown
+ *
+ */
+@Configuration
+public class CorsConfig {
 
-        @Bean
-        public WebMvcConfigurer corsConfigurer(){
-            return new WebMvcConfigurer(){
+	@Bean
+	public WebMvcConfigurer corsConfigurer() {
+		return new WebMvcConfigurer() {
 
-                @Override
-                public void addCorsMappings(CorsRegistry registry){
+			@Override
```

Comment:
This method should have its own JavaDoc, including what it accepts as input and what it does with it.